opbeat/base.py

```diff
@@ -384,14 +359,9 @@ def capture(self, event_type, data=None, date=None, api_path=None,
 
         data = self.build_msg_for_logging(event_type, data, date,
                                           extra, stack, **kwargs)
-
-        if not api_path:
-            api_path = defaults.ERROR_API_PATH
-
-        data['servers'] = [server+api_path for server in self.servers]
-        self.send(**data)
-
-        return data['client_supplied_id']
+        if data:
+            self.send(**data)
+            return data['errors'][0]['id']
```

Comment:
Yes, if `data` isn't `None`, there will always be at least one error object (for now, also maximum one, we haven't added queueing for errors yet)